pkg/apis/core/v1/appconfiguration.go

```diff
@@ -20,61 +20,61 @@ type Accessory map[string]interface{}
 // import models.schema.v1.monitoring as m
 // import models.schema.v1.database as d
 //
-//		helloWorld: ac.AppConfiguration {
-//		   # Built-in module
-//		   workload: wl.Service {
-//		       containers: {
-//		           "main": c.Container {
-//		               image: "ghcr.io/kusion-stack/samples/helloworld:latest"
-//		               # Configure a HTTP readiness probe
-//		               readinessProbe: p.Probe {
-//		                   probeHandler: p.Http {
-//		                       url: "http://localhost:80"
-//		                   }
-//		               }
-//		           }
-//		       }
-//		   }
+//	  helloWorld: ac.AppConfiguration {
+//	      # Built-in module
+//	      workload: wl.Service {
+//	          containers: {
+//	              "main": c.Container {
+//	                  image: "ghcr.io/kusion-stack/samples/helloworld:latest"
+//	                  # Configure a HTTP readiness probe
+//	                  readinessProbe: p.Probe {
+//	                      probeHandler: p.Http {
+//	                          url: "http:localhost:80"
```

Comment:
Here seems to be a typo? `url: "http:localhost:80"` ---> `url: "http://localhost:80"`